src/@next/components/organisms/ProductList/types.ts

```diff
@@ -1,9 +1,7 @@
 import { ProductList_products_edges_node } from "@saleor/sdk/lib/queries/gqlTypes/ProductList";
 
-type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
-
 export interface IProps {
-  products: Array<PartialBy<ProductList_products_edges_node, "category">>;
+  products: Array<Partial<ProductList_products_edges_node>>;
```

Comment:
I don't get why did you make this change, is it some part of code cleanup?

Reply:
`ProductList_products_edges_node` doesn't contain `category` anymore.